guides/doc-Release_Notes/topics/foreman.adoc

```diff
@@ -4,7 +4,14 @@
 [id="foreman-headline-features"]
 == Headline Features
 
+ifndef::foreman-deb[]
 There are no highlights with Foreman {ProjectVersion}.
+endif::[]
+ifdef::foreman-deb[]
+=== Running Foreman on Debian 12 (Bookworm)
+
+Foreman now supports running on Debian 12 (Bookworm).
```

Comment:
I'd like to support it with Foreman 3.12 now that puppetserver is available to give users a better upgrade path. You already made sure we have packages so it should be easy.

Reply:
should this be read: "drop the release note here and add it in 3.12" or "keep it here but *also* pick it to 3.12"?

Reply:
I'm leaning to the latter: introduce it as a headline feature here, but also note it for the future patch releases. That way users who read them as a patch release know they can benefit from it. My reasoning behind that is that I wouldn't read headline features when I upgrade a patch release because I'd assume they don't change.

Reply:
Sounds like no changes are needed from me, cool!